Program.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+namespace AppBackendTraining
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();
```

Comment:
You might want to attempt logging exception in case this start fails ^^
(Bit of a challenge)